Section 4 - Terraform Modules & Workspaces/module-outputs/projects/C/my-sg.tf

```diff
@@ -3,11 +3,40 @@ module "sgmodule" {
 }
 
 resource "aws_instance" "web" {
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure IMDSv1 is disabled for AWS EC2 instances**
 
**Vulnerability**
AWS EC2 instances that have IMDSv1 enabled are vulnerable to server side request forgery (SSRF) attacks.
 
**Remediation**
In AWS Console - 
 1. When launching a new instance in the Amazon EC2 console, select the following options on the Configure Instance Details page:
     a. Under Advanced Details, for Metadata accessible, select Enabled.
     b. For Metadata version, select V2.

source: tenablecs-